.github/workflows/issues2md.yml

```diff
@@ -0,0 +1,29 @@
+# Action to backup issues using gh2md, contributed by
+# https://github.com/0ut0fcontrol.
+# See https://github.com/mattduck/gh2md/issues/11.
+name: Issues2Markdown
+on:
+  schedule:
```

Comment:
Is there a benefit to also making this a manually-started workflow, like the deployment workflow?

Reply:
Good point.  Another item for me to google really quick.

Reply:
OK...committed that change as well.  Now I can just run it on-demand.